developer_setup.md

```diff
@@ -83,17 +83,34 @@ As per 20+, with the following changes:
   brew install git
   brew install memcached
   brew install postgresql
+  brew install graphviz
```

Comment:
I know this helps bundler in printing their pretty graphs, but it is optional and not really necessary for our product